iohk-monitoring/iohk-monitoring.cabal

```diff
@@ -130,7 +133,7 @@ library
   if flag(performance-test-queue)
     cpp-options:       -DPERFORMANCE_TEST_QUEUE
 
-  ghc-options:         -Wall -Werror
```

Comment:
Why did you remove `-Werror`? It helps us to keep the code as clean as possible, no?

Reply:
oh, I should readd it.